README.rdoc

```diff
@@ -31,7 +31,7 @@ A small example usage:
 == Build Status
 
 {<img src="https://travis-ci.org/ged/ruby-pg.svg?branch=master" alt="Build Status Travis-CI" />}[https://travis-ci.org/ged/ruby-pg]
-{<img src="https://ci.appveyor.com/api/projects/status/at4g4swb2cd4xji7/branch/master?svg=true" alt="Build Status Appveyor" />}[https://ci.appveyor.com/project/ged/ruby-pg]
+{<img src="https://ci.appveyor.com/api/projects/status/gjx5axouf3b1wicp?svg=true" alt="Build Status Appveyor" />}[https://ci.appveyor.com/project/ged/ruby-pg]
```

Comment:
I see the badge, but the link says `Project not found or access denied.`

Reply:
Ah I see what I did. I couldn't convert the old Appveyor project to use Github, so I had to delete and re-create and that gave the project link a suffix. Pushing a fix.